app/models/task.rb

```diff
@@ -102,6 +105,14 @@ def self.ransackable_associations(_auth_object = nil)
     %w[labels]
   end
 
+  def push_to_nbp
+    NbpPushJob.perform_later self if Settings.nbp&.push_connector&.enable
+  end
+
+  def remove_from_nbp
+    NbpDeleteJob.perform_later uuid if Settings.nbp&.push_connector&.enable
+  end
```

Comment:
We also need to re-consider the visibility level for pushing tasks. Currently, all tasks are pushed regardless of their visibility, aren't they? However, private tasks should remain private.

Reply:
Done :+1: